.travis/preinstall.sh

```diff
@@ -46,5 +46,17 @@ if [ -n "$USE_ROCKSDB" ]; then
     sudo make install-shared
 fi
 
+if [-n "$USE_LOG4CPP" ]; then
```

Comment:
This won't work. The variable will never be set because it does not come from CMAKE. These environment variables come from .travis.yml.

See my comment regarding that we don't actually build with LOG4CPLUS at all in CI.

Reply:
THANKS! I will remove it from the build.